helm_deploy/values-dev.yaml

```diff
@@ -12,6 +12,14 @@ generic-service:
     INDEX_COMPLETE_THRESHOLD: 500000
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
 
+  # Additional allowlist rules
```

Comment:
This is open to internet - not needed

Reply:
No it's not, it is IP allow-listed  - see comment, refer to charts default `values.yaml`